### DISCUSSION TITLE
How to run Blockchain like Ganache in Hardhat

### DISCUSSION THREAD

**Discussion by vkpatva** [ConsenSys-archive/ganache#4550] [General]
While working with Truffle & Ganache we can continue the state of Blockchain I mean Ganache stores the deployed smart contract and other transactions , since truffle will be discontinued soon, I have a question regarding statefullness of Hardhat Node.

**Comment by davidmurdoch** [comment]
I don't think there is a complete Ganache replacement (which is why Consensys didn't publish a migration guide). Foundry's Anvil is probably closest thing you'll get to Ganache right now.
